src/test/kotlin/com/karumi/todoapiclient/MockWebServerTest.kt

```diff
@@ -10,100 +10,87 @@ import org.junit.Assert
 import org.junit.Assert.assertEquals
 import org.junit.Before
 import java.io.File
-import java.io.IOException
 
 open class MockWebServerTest {
 
-    private var server: MockWebServer? = null
+    private lateinit var server: MockWebServer
 
     @Before
-    @Throws(Exception::class)
     open fun setUp() {
         this.server = MockWebServer()
```

Comment:
uhmm you can not instanciate server inline?
private val server = MockWebServer() ??

or if you instanciate in this way the mockwebserver store old references?

Reply:
I think it cleans everything up on the `shutdown` method so no need to create it here just as you said, I'm moving the assignment to the property declaration 👌